src/cmr/metadata/proxy/concepts/granule.clj

```diff
@@ -112,17 +111,39 @@
     (and (not (:inherited link-data))
          (= const/datafile-link-rel rel))))
 
-(defn extract-datafile-link
+(def OPENDAP_LOWERCASE
```

Comment:
what the ...

Reply:
Sir, I beg of you to leave your Bash at the door before entering this fine establishment ...